az-cvm-vtpm/src/vtpm/mod.rs

```diff
@@ -165,7 +185,38 @@ pub fn get_quote(data: &[u8]) -> Result<Quote, QuoteError> {
     let signature = rsa_sig.signature().to_vec();
     let message = attest.marshall()?;
 
-    Ok(Quote { signature, message })
+    // Drop the context because access to the tpm device is again needed in the `read_all_pcrs` function.
+    drop(context);
```

Comment:
I don't think we to recreate the context to perform the reads. if we remove the sessions it should work: `context.clear_sessions();`